tasklib/tests.py

```diff
@@ -1111,40 +1120,43 @@ def test_non_unicode_task(self):
         Task(self.tw, description="test task").save()
         self.tw.tasks.get()
 
+
 class ReadOnlyDictViewTest(unittest.TestCase):
 
     def setUp(self):
-        self.sample = dict(l=[1,2,3], d={'k':'v'})
+        self.sample = dict(sample_list=[1, 2, 3], sample_dict={'key': 'value'})
         self.original_sample = copy.deepcopy(self.sample)
         self.view = ReadOnlyDictView(self.sample)
 
     def test_readonlydictview_getitem(self):
-        l = self.view['l']
-        self.assertEqual(l, self.sample['l'])
+        sample_list = self.view['sample_list']
+        self.assertEqual(sample_list, self.sample['sample_list'])
 
         # Assert that modification changed only copied value
-        l.append(4)
-        self.assertNotEqual(l, self.sample['l'])
+        sample_list.append(4)
+        self.assertNotEqual(sample_list, self.sample['sample_list'])
 
         # Assert that viewed dict is not changed
         self.assertEqual(self.sample, self.original_sample)
 
     def test_readonlydictview_contains(self):
-        self.assertEqual('l' in self.view, 'l' in self.sample)
-        self.assertEqual('d' in self.view, 'd' in self.sample)
-        self.assertEqual('k' in self.view, 'k' in self.sample)
+        self.assertEqual('sample_list' in self.view,
+                         'sample_list' in self.sample)
+        self.assertEqual('sample_dict' in self.view,
+                         'sample_dict' in self.sample)
+        self.assertEqual('key' in self.view, 'key' in self.sample)
 
         # Assert that viewed dict is not changed
         self.assertEqual(self.sample, self.original_sample)
 
     def test_readonlydictview_iter(self):
-        self.assertEqual(list(k for k in self.view),
-                         list(k for k in self.sample))
+        self.assertEqual(list(key for key in self.view),
+                         list(key for key in self.sample))
 
         # Assert the view is correct after modification
         self.sample['new'] = 'value'
-        self.assertEqual(list(k for k in self.view),
-                         list(k for k in self.sample))
+        self.assertEqual(list(key for key in self.view),
+                         list(key for key in self.sample))
```

Comment:
I'd probably opt to separate these arguments to `assertEqual` out onto their own lines, as they are a little bit hard to parse as-is with the `list()` calls in there as well.

Reply:
Fixed in 8f4fabf